packages/ui/cypress/support/page-objects/sendTxModal.ts

```diff
@@ -1,6 +1,14 @@
 export const sendTxModal = {
   sendTxTitle: () => cy.get('[data-cy=title-send-tx]'),
-  fieldTo: () => cy.get('[data-cy=field-to]'),
-  fieldAmount: () => cy.get('[data-cy=field-amount]'),
-  buttonSend: () => cy.get('[data-cy=button-send]')
+  sendTokensFieldTo: () => cy.get('[data-cy=send-tokens-field-to]'),
+  sendTokensFieldAmount: () => cy.get('[data-cy=send-tokens-field-amount]'),
+  buttonSend: () => cy.get('[data-cy=button-send]'),
+  setIdentitySection: () => cy.get('[data-cy=section-set-identity]'),
+  setIdentityField: (field: string) => cy.get(`[data-cy=set-identity-field-${field}]`),
+  setIdentityFieldInput: (field: string) => cy.get(`[data-cy=input-set-identity-field-${field}]`),
+  setIdentityFieldElement: (field: string, element = 'label') =>
```

Comment:
```suggestion
  setIdentityFieldLabel: (field: string, element = 'label') =>
```

Reply:
Nope, because is could be 'label', or 'div' for instance

Reply:
we use both.